python/tchannel/messages/base.py

```diff
@@ -25,3 +30,59 @@ def __str__(self):
             str(self.__class__.__name__),
             ", ".join(attrs)
         )
+
+    @staticmethod
+    def compute_checksum(checksum_type, args, csum=0):
+        if csum is None:
+            csum = 0
+
+        if checksum_type == ChecksumType.none:
+            return None
+        elif checksum_type == ChecksumType.crc32:
+            for arg in args:
+                csum = zlib.crc32(arg, csum)
+        # TODO investigate farm32 cross platform issue
+        elif checksum_type == ChecksumType.farm32:
+            for arg in args:
+                csum = farmhash.hash32withseed(arg, csum)
```

Comment:
Upon a second look, it appears that the Python implementation doesn't provide that method. Throw a `NotImplementedError` here for now.